api/v1alpha1/jwt_types.go

```diff
@@ -52,8 +52,16 @@ type JWTProvider struct {
 	// For examples, following config:
 	// The claim must be of type; string, int, double, bool. Array type claims are not supported
 	//
+	// +optional
 	ClaimToHeaders []ClaimToHeader `json:"claimToHeaders,omitempty"`
 
+	// RecomputeRoute clears the route cache and recalculates the routing decision.
+	// This field must be enabled if the headers generated from the claim are used for
+	// route matching decisions.
+	//
+	// +optional
+	RecomputeRoute *bool `json:"recomputeRoute,omitempty"`
```

Comment:
Should we verify that when recomputeRoute is enabled, the length of claimToHeaders is not zero?

Reply:
yes !

Reply:
can you update cel validation?

Reply:
done ! thanks @zhaohuabing for helping me with my first CEL expression :)